dataflux_core/tests/fake_gcs.py

```diff
@@ -56,9 +59,12 @@ def list_blobs(
                     results.append(self.blobs[name])
         return results
 
-    def blob(self, name: str):
+    def blob(self, name: str, missing_path: bool = False):
```

Comment:
There's no reason to pass this in as a KW arg here, it should just be defined outside of the function signature in the body.

Reply:
I changed it to default argument since, it will mostly be False and very rarely it will be true. Also the default value is static and is not determined based on any other aqrgument. Can you explain your reasoning on why it should be in function compared to default arg ?

Reply:
You set the value for it based on the "name" input at 63. The correct way to format this would be to set the variable `missing_path=False` within the function and allow it to be altered by line 63. There's just no need for that to be set as a function parameter, and it's misleading given that line 63 will override that value. I would recommend doing something like this:
```
def blob(self, name: str):
        missing_path = False
        if name == "missing-path":
            missing_path = True
        if name not in self.blobs:
            self.blobs[name] = Blob(
                name, bucket=self, missing_bucket=missing_path)
        return self.blobs[name]
```